install/kubernetes/templates/clusterrole.yaml

```diff
@@ -44,6 +44,7 @@ rules:
     resourceNames:
       - tracingpolicies.cilium.io
       - tracingpoliciesnamespaced.cilium.io
+      - tetragonpods.cilium.io
```

Comment:
we'll make our final decision for the CRD name in https://github.com/cilium/tetragon/pull/1281. this is fine for this PR.

Reply:
ok